sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_output_labeled_tables_async.py

```diff
@@ -0,0 +1,138 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_differentiate_output_labeled_tables_async.py
+
+DESCRIPTION:
+    This sample demonstrates the differences in output that arise when begin_recognize_custom_forms
+    is called with custom models trained with fixed vs. dynamic table tags.
+    The models used in this sample can be created in the sample_train_model_with_labels.py using the
+    training files in
+    https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_forms/labeled_tables
+
+    Note that Form Recognizer automatically finds and extracts all tables in your documents whether the tables
+    are tagged/labeled or not. Therefore, you don't have to label every table from your form with a table tag and your
+    table tags don't have to replicate the structure of every table found in your form. Tables extracted
+    automatically by Form Recognizer will be included in the `tables` property under `RecognizedForm.pages`.
+
+    A conceptual explanation of using table tags to train your custom form model can be found in the
+    service documentation: https://docs.microsoft.com/azure/cognitive-services/form-recognizer/supervised-table-tags
+
+USAGE:
+    python sample_differentiate_output_labeled_tables_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+    3) MODEL_ID_FIXED_ROW_TABLES - the ID of your custom model trained with labels on fixed row tables
+    4) MODEL_ID_DYNAMIC_ROW_TABLES - the ID of your custom model trained with labels on dynamic row tables
+"""
+
+import os
+import asyncio
+
+
+class TestDifferentiateOutputLabeledTablesAsync(object):
+
+    async def test_recognize_tables_fixed_rows_async(self):
+        from azure.core.credentials import AzureKeyCredential
```

Comment:
Is this sample runnable?

Reply:
yes

Reply:
hmm are the model ids created every time the sample is run? or how does the magic work?
Not blocking your PR, just curious

Reply:
No, these are static model IDs that get passed in as environment variables. All samples are run from an existing FR resource, nothing created on the fly here.

Reply:
I worry someone might delete the models and then all samples stop working, but it will be pretty clear, so easy to fix.
of course, if it ever becomes a problem the samples can be updated an in charge of creating the model too